app/storage/metadata_parser.py

```diff
@@ -26,6 +26,24 @@ def string_parser(string_value):
         raise TypeError('Claim not a valid string value')
     return string_value
 
+def numeric_string_parser(to_parse, length=None):
```

Comment:
To keep it consistent with the other method?
`to_parse` > `string_value`?, or `string_to_parse`.